packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -296,11 +296,20 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       // Allow the caller to handle the key down
       onKeyDown?.(ev);
 
-      // Handle copy if focus is in the selected items list
+      // (I'm attempting to send a draft PR to see what it looks like if we violate lint,
+      // and get feedback on my first checkin).
+      // The explicit keycodes is probably the localization issue mentioned below
+      const isDel = ev.which === KeyCodes.del;
+      const isCut = (ev.shiftKey && isDel) || (ev.ctrlKey && ev.which === KeyCodes.x);
```

Comment:
Why is shift delete = cut? (Is this a mistake or a cut command I didn't know about?)

Reply:
Shift+Del is one of the standard keyboard shortcuts for the Cut operation, and is how I first learned to Cut. It functions identically to Ctrl+X.

Reply:
Shift+Insert is the same as Ctrl+V (two ways to Paste :)

Reply:
I did not know that. Interesting